build-tools/packages/build-cli/src/commands/exec.ts

```diff
@@ -19,6 +19,7 @@ export default class ExecCommand extends PackageCommand<typeof ExecCommand> {
 	};
 
 	protected async processPackage(pkg: Package): Promise<void> {
+		// eslint-disable-next-line no-warning-comments
```

Comment:
If we have our linter configured to warn about TODOs, we probably want warnings for the and shouldn't suppress those warnings.

If we don't want those warnings, we should fix the todos, or change the linter settings.

Reply:
Yeah, these are left as warnings intentionally, so don't suppress.